### PR TITLE
Updated B5 guidance for .modal-header and .close

### PR DESCRIPTION
**Pull request by orangejenny**
## Technical summary
Combine the guidance on `.close` and `.modal-header`, which occur in the code together, rather than having the tool mark them as two separate issues.

### Safety story
Only affects documentation and an internal engineering tool.

### Automated test coverage

The migration tool has coverage.

### QA Plan

no

### Rollback instructions
- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Comment by orangejenny** [comment]
Also added a minor change to the migration tool's prompting.

File: corehq/apps/hqwebapp/utils/bootstrap/changes_guide/css-modal-header.md [hunk 1]
```diff
@@ -1,25 +0,0 @@
-The `modal` component was rewritten and now the close button should come after
-the `modal-title`. Also, there is a new accessibility attribute `aria-labelledby`
-which you can add to label the modal with the title. See documentation for details...
-
-An EXAMPLE for how to apply this change is provided below.
-Please see docs for further details.
-
-Previously:
-```
-<div class="modal-header">
```
Comment (jingcheng16) [inline corehq/apps/hqwebapp/utils/bootstrap/changes_guide/css-modal-header.md:10]: Do you mind replace the example in `css-close` with the example here?

Comment (orangejenny) [reply corehq/apps/hqwebapp/utils/bootstrap/changes_guide/css-modal-header.md:10]: Sure, updated in 95044f2c751

**Review by jingcheng16 — APPROVED** [review]
Thank you!

**Pull request merged**